deployments/examples/ocis_wopi/config/onlyoffice/entrypoint-override.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+# we can't mount it directly because the run-document-server.sh script wants to move it
```

Comment:
I assume it wasn't possible to change `run-document-server.sh` script? This will get very messy when we have a lot of sh scripts that all do "something"